.vscode/settings.json

```diff
@@ -6,13 +6,8 @@
   "[json]": {
     "editor.defaultFormatter": "denoland.vscode-deno"
   },
-  "debug.javascript.defaultRuntimeExecutable": {
-    "pwa-node": "${workspaceFolder}/.vscode/mise-tools/node"
-  },
-  "deno.path": "${workspaceFolder}/.vscode/mise-tools/deno",
```

Comment:
My [upstream PR](https://github.com/denoland/vscode_deno/pull/1245#issuecomment-2614807459) was denied, I'm just going to back away from using this type of configuration.